airbyte-integrations/connectors/destination-bigquery/metadata.yaml

```diff
@@ -2,7 +2,7 @@ data:
   connectorSubtype: database
   connectorType: destination
   definitionId: 22f6c74f-5699-40ff-833c-4a879ea40133
-  dockerImageTag: 1.5.0
+  dockerImageTag: 1.4.5
```

Comment:
Nit: In the future, it's probably better to add a 'pin' for each registry (cloud and oss) than change the 'latest version that exists', e.g: https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/source-salesforce/metadata.yaml#L14-L18